fcrepo4/fcrepo4
# Sets proper actor values for emitted messages.

The actor value is now populated with the user URI derived from the userId based on changes made in fcrepo-2608.

**JIRA Ticket**: https://jira.duraspace.org/browse/FCREPO-2603

* Other Relevant Links (Mailing list discussion, related pull requests, etc.)
https://jira.duraspace.org/browse/FCREPO-2608
https://jira.duraspace.org/browse/FCREPO-2628

# How should this be tested?

Unit tests cover the serialization and deserialization processes. If you would like to verify the contents of the generated messages you can use the following wiki to view the events generated by fedora:
https://wiki.duraspace.org/display/FEDORA4x/How+to+Inspect+Event+Messages+Generated+by+Fedora
If you do not use the -Dfcrepo.auth.webac.userAgent.baseUri flag the actor value should appear as "info:fedora/local-user#<username>".

# Additional Notes:
Any additional information that you think would be helpful when reviewing this PR.

* possible documentation updates.
